### PR TITLE
Proposed changes to CLI to support manual testing

### PR DESCRIPTION
**Pull request by jvanderhoof**
This is a proposed interface change to allow this project to support common customer workflows. The goal of this interface is to simplify manual testing of a wide variety of customer scenarios to ensure product integrity.

Upgrades for a single master and an auto-failover cluster have been validated as follows:

#### Single master upgrade

```
bin/dap --provision-master --version 11.5.0
bin/api --load-policy-and-values
bin/dap --upgrade-master 5.11.0
```

#### Auto-failover cluster (w/ follower) upgrade

```
bin/dap --provision-master --version 11.5.0
bin/api --load-policy-and-values
bin/dap --provision-standbys --version 11.5.0
bin/dap --enable-auto-failover
bin/dap --provision-follower --version 11.5.0
bin/api --authenticate-user

bin/dap --upgrade-master 5.11.0
bin/dap --provision-standbys --version 5.11.0
bin/dap --enable-auto-failover
bin/dap --provision-follower --version 5.11.0
bin/api --authenticate-user
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,11 +12,89 @@ Tools and scripts  utilities that make it easier to make, manage, and run demos
...
+```sh
+$ bin/dap --provision-master
+$ bin/dap --provision-standbys
+$ bin/dap --enable-auto-failover
+```
+
+Given the above, to add a follower (behind L7 load balancer), add data, and validate retrieval:
+```sh
+$ bin/dap --provision-follower
+$ bin/api --load-policy-and-values
```
Comment (h-artzi) [inline README.md:32]:
```
 bin/api --load-policy-and-values
{"created_roles":{},"version":3}curl: (35) OpenSSL SSL_connect: SSL_ERROR_SYSCALL in connection to conjur-master.mycompany.local:443
➜  dap-intro git:(cli-workflows) bin/api --fetch-secrets
--fetch-secrets is not a valid option
```

**Review by h-artzi — CHANGES_REQUESTED** [review]

File: bin/dap [hunk 2]
```diff
@@ -299,79 +271,71 @@ function _upgrade_master_via_backup_restore {
...
+    --promote-standby ) CMD='_perform_promotion' ; shift ;;
+    --import-custom-certificates ) CMD='_import_certificates' ; shift ;;
+    --provision-standbys ) CMD='_setup_standbys' ; shift ;;
+    --enable-auto-failover ) CMD='_enable_autofailover' ; shift ;;
+    --provision-follower ) CMD='_setup_follower' ; shift ;;
+    --upgrade-master ) shift ; CMD="_upgrade_via_backup_restore $1" ; shift ;;
+    --version ) shift ; TAG=$1 ; shift ;;
+    --dry-run ) DRY_RUN=true ; shift ;;
+    -h | --help ) _print_help ; shift ;;
+    # --disable-auto-failover ) CMD='_disable_autofailover' ; shift ;;
```
[the remark this answers is not in the document]
Comment (jvanderhoof) [reply bin/dap:330]: I'm going to remove this.  Disabling auto-failover is fraught with peril and doesn't result in auto-failover functionality being fully removed. The script un-enrolls ETCD nodes, which gets us to the point where we can upgrade.

File: README.md [hunk 3]
```diff
@@ -69,6 +147,7 @@ Usage: bin/dap cluster [options]
 
     --create-backup             Generates a backup of the Master. The backup can be found in the system/backup folder
     --dry-run                   Displays the commands that will be run, without actually running them
+    --disable-auto-failover     Removes nodes into and auto-failover cluster
```
[the remark this answers is not in the document]
Comment (jvanderhoof) [reply README.md:150]: great catch!

File: system/configuration/certificates/ca-chain.cert.pem [hunk 4]
```diff
@@ -0,0 +1,100 @@
+-----BEGIN CERTIFICATE-----
```
[the remark this answers is not in the document]
Comment (jvanderhoof) [reply system/configuration/certificates/ca-chain.cert.pem:1]: I've filed issue #73 to capture this work.

File: README.md [hunk 5]
```diff
@@ -69,6 +147,7 @@ Usage: bin/dap cluster [options]
 
     --create-backup             Generates a backup of the Master. The backup can be found in the system/backup folder
     --dry-run                   Displays the commands that will be run, without actually running them
```
[the remark this answers is not in the document]
Comment (jvanderhoof) [reply README.md:149]: Good catch. I'm going to pull all that out of the README.

File: README.md (hunk 1 above)
Comment (jvanderhoof) [reply README.md:32]: Added 😄 .  I also updated the Readme

**Comment by h-artzi** [comment]
fetch secrets is good, but a simple add on might be a set secrets as well. To ensure they don't return empty strings.

**Comment by jvanderhoof** [comment]
@h-artzi, this tool is intended to simplify common workflows (for engineers).  I don't want to replace the Conjur CLI.  Is there anything else this needs, or can we merge these improvements?

File: README.md [hunk 6]
```diff
@@ -87,8 +87,10 @@ $ bin/api --fetch-secrets
 |--against-master|configuration|Runs read actions against the master||
 |--authenticate-user|action|• Authenticates with default user and password<br>• Displays the resulting authentication token||
 |--fetch-secrets|action|• Authenticates<br>• Retrieves variable values|Run against follower unless `--against-master` flag is present|
-|--load-policy-and-values|action|• Authenticates<br>• Loads policy and variable values|Run against master|
-|--password `<password>`|configuration|Uses a non-default password for authentication||
+|--load-policy|action|• Authenticates<br>• Loads policy|Run against master|
+|--load-policy-and-values|action|• Authenticates<br>• Loads policy and variable values|Run against master, equivalent to running '--load-policy' and '--set-secrets'|
+|--password `<password>`|configuration|Uses a non-default password for authentication|• Authenticates<br>• Sets variable values|Requires `--load-policy` before running|
```
Comment (h-artzi) [inline README.md:92]: Authenticates<br>• Sets variable is in the wrong row. Need to be moved down one

Comment (jvanderhoof) [reply README.md]: fixed

**Review by h-artzi — APPROVED** [review]

**Pull request merged**